src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Diagnostics/BindingDiagnostics.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+// Description:
+//  Binding diagnostics API
+//
+
+using System.Collections.Concurrent;
```

Comment:
Unused, I think.  If so, delete it.

Reply:
Good catch, it was my final change that removed the ConcurrentQueue and just went with a lock to protect pendingEvents.

---
In reply to: [488320841](https://github.com/dotnet/wpf/pull/3505#discussion_r488320841) [](ancestors = 488320841)